.github/workflows/visual-regression.yml

```diff
@@ -0,0 +1,24 @@
+name: Visual Regression
```

Comment:
You won't need this file unless you want to add Happo example tests in addition to your Cypress screenshots.